tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -40,6 +40,27 @@ module.exports = {
       )
     },
 
+    /**
+     * Return first elementID that matches given selector and is visible
+     *
+     * @param {string} using
+     * @param {string} value
+     */
+    getVisibleElementID: async function (using, value) {
```

Comment:
would that be useful also in other places? maybe put it into a custom command?

Reply:
`CustomCommands` need to return itself. So, we would need to use the callback function instead. If we need it elsewhere, we can think of it then(following YAGNI).